Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -114,6 +117,7 @@ public function getFacets(): Facet
         try {
             $this->populateFilters($includeFilters);
         } catch (Exception $e) {
+            $this->debugWithSource(sprintf('Cannot populate filters, error message: %e', $e));
```

Comment:
@olsi-qose what is this pattern btw? Why do we first log the message and then the exception?